app/views/layouts/application.html.erb

```diff
@@ -3,12 +3,14 @@
   <head>
     <title>Paps</title>
     <%= csrf_meta_tags %>
-
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <% flash.each do |key, value| %>
+      <div class="alert alert-<%= key %>"><%= value %></div>
+    <% end %>
     <%= yield %>
   </body>
 </html>
```

Comment:
this should be converted to haml or else we will have a merge conflict

Reply:
Good catch, will do!